### DISCUSSION TITLE
Welcome to 'statemachine' Discussions!

### DISCUSSION THREAD

**Discussion by gemerden** [gemerden/statemachine#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ksaaeu** [comment]
Just started to use it -> awsome, great job, thank you

**Reply by gemerden** [reply]
How is it working out for you?
